app-engine/src/main/java/be/cytomine/appengine/controllers/TaskRunController.java

```diff
@@ -58,7 +58,7 @@ public ResponseEntity<?> provisionJson(
         @PathVariable("param_name") String parameterName,
         @RequestBody JsonNode provision
     ) throws ProvisioningException {
-        log.info("/task-runs/{run_id}/input-provisions/{param_name} JSON PUT");
+        log.info("/task-runs/%s/input-provisions/%s JSON PUT", runId, parameterName);
```

Comment:
```suggestion
        log.info("/task-runs/{}/input-provisions/{} JSON PUT", runId, parameterName);
```

I think it is better to use SLF4J's parameterized messaging